tenable/io/v3/vm/exclusions/iterator.py

```diff
@@ -0,0 +1,19 @@
+'''
```

Comment:
File will be deleted if we use SearchIterator and CSVChunkIterator directly

Reply:
done.